api/getIPs.php

```diff
@@ -0,0 +1,65 @@
+<?php 
+/**
+ * This code takes a domain name or subnet as the argument and returns all
+ * IP addresses and names associated with the domain or subnet.
+ * Required parameters are api name, api token, and domain
+ * domain can be domain name or a dotted quad subnet. (10.10.10.0)
+ *
+ * http://phpipam/api/getIPs.php?apiname=[name]&apitoken=[token]&domain=[domain name]
+ */
+
+include_once '../config.php';
+include_once 'tokenValid.php';
+
+# Setup variables from request
+$domain = $_REQUEST['domain'];
+$api_app = $_REQUEST['apiapp'];
+$api_token = $_REQUEST['apitoken'];
+
+# Check the api_app name and token passed to us
+$validate = ValidateToken($api_app, $api_token);
+if ($validate !== true) {
+  print "$validate";
+  exit();
+}
+
+$data = new mysqli("localhost", $db['user'], $db['pass'], $db['name']);
+
+if ($data->connect_errno) {
+  printf("Connect failed: %s\n", $data->connect_error);
+  exit();
+}
+
+# Check if we are doing a reverse zone
+$isIP = filter_var($domain, FILTER_VALIDATE_IP);
+
+if ($isIP) {
+
+  $subnetip = ip2long($domain);
+  $subnetsql = "SELECT id FROM subnets where subnet=$subnetip";
```

Comment:
You're using a direct SQL query here, but you used a prepared statement in `getFreeIP.php` on line 52.  I tend to favor the prepared statements, unless there's a compelling reason not to do so.